server/core/src/https/views/admin/groups.rs

```diff
@@ -0,0 +1,617 @@
+    members: Vec<MemberInfo>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+struct GroupACP {
+    enabled: bool,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+struct MemberInfo {
+    uuid: String,
+    name: String,
+    spn: String,
+    displayname: String,
+}
+
+#[derive(Template)]
+#[template(path = "admin/admin_overview.html")]
+struct GroupCreateView {
+    access_info: AccessInfo,
+    partial: GroupCreatePartialView,
+}
+
+#[derive(Template)]
+#[template(path = "admin/admin_group_create_partial.html")]
+struct GroupCreatePartialView {
+    can_rw: bool,
+    groups: Vec<GroupInfo>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub(crate) struct GroupCreateFormData {
+    name: String,
+    guid: Option<String>,
+    managed_by: Option<String>,
+}
+
+pub(crate) async fn view_group_create_get(
+    State(state): State<ServerState>,
+    HxRequest(is_htmx): HxRequest,
+    Extension(kopid): Extension<KOpId>,
+    VerifiedClientInformation(client_auth_info): VerifiedClientInformation,
+) -> axum::response::Result<Response> {
+    let can_rw = get_can_rw(&state, &kopid, &client_auth_info).await?;
+
+    let groups = get_groups_info(state, &kopid, client_auth_info).await?;
+    let groups_partial = GroupCreatePartialView { can_rw, groups };
+
+    let push_url = HxPushUrl(Uri::from_static("/ui/admin/group/create"));
+    Ok(if is_htmx {
+        (push_url, HtmlTemplate(groups_partial)).into_response()
+    } else {
+        (
+            push_url,
+            HtmlTemplate(GroupCreateView {
+                access_info: AccessInfo::new(),
+                partial: groups_partial,
+            }),
+        )
+            .into_response()
+    })
+}
+
+pub(crate) async fn view_group_delete_post(
+    State(state): State<ServerState>,
+    HxRequest(is_htmx): HxRequest,
+    Extension(kopid): Extension<KOpId>,
+    Path(group_uuid): Path<Uuid>,
+    VerifiedClientInformation(client_auth_info): VerifiedClientInformation,
+) -> axum::response::Result<Response> {
+    dbg!(group_uuid);
+    view_groups_get(
+        State(state),
+        HxRequest(is_htmx),
+        Extension(kopid),
+        VerifiedClientInformation(client_auth_info),
+    )
+    .await
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub(crate) struct GroupDetailsFormData {
+    name: String,
+    spn: String,
+    entry_manager: String,
+}
+
+#[derive(Template)]
+#[template(path = "admin/admin_group_details_partial.html")]
+struct GroupMemberDetailsPartialView {
+    group: GroupInfo,
+    can_edit: bool,
+}
+
+pub(crate) async fn view_group_save_post(
+    State(state): State<ServerState>,
+    HxRequest(_is_htmx): HxRequest,
+    Extension(kopid): Extension<KOpId>,
+    VerifiedClientInformation(client_auth_info): VerifiedClientInformation,
+    Path(guuid): Path<Uuid>,
+    Form(data): Form<GroupDetailsFormData>,
+) -> axum::response::Result<Response> {
+    let filter = filter_all!(f_and!([f_eq(Attribute::Class, EntryClass::Group.into())]));
```

Comment:
For now, let's replace this with a todo!() or a 500 error or something as we need to rework this a bit, and it'll help to keep it clearer.